README.md

```
@@ -1 +1,153 @@
+ Professores
+
+ Funcionalidades administrativas e pedagógicas
+
+O sistema foi desenvolvido para melhorar a organização, a acessibilidade e a eficiência da gestão escolar.
+
+----
+
+
+##  Tecnologias Utilizadas
+
+Next.js	       Framework React para aplicações web modernas
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 8

(MD010, no-hard-tabs)